cla-backend-go/v2/cla_manager/service.go

```diff
@@ -518,6 +520,13 @@ func (s *service) CreateCLAManagerRequest(contactAdmin bool, companyID string, p
 		return nil, ErrNotInOrg
 	}
 
+	// Check if user has organization
+	if lfxUser.Account.Name == "" {
```

Comment:
Is this how we check that the specified user is not associated with any organization?

Reply:
So on checking the Lfx User Model this is the association with company via the Account attribute. I could be wrong